src/main/java/com/google/sps/servlets/PhotoShoppingServlet.java

```diff
@@ -29,21 +32,31 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // TODO: Based on request.getParameter("photo-category"), call methods from photo detection
-    // classes,
-    // passing request.getParameter("blob-key") as argument. These methods build the shopping query
-    // and call
-    // the {@code getShoppingResultsPage} method from GoogleShoppingResultsWrapper.
+    // classes, passing request.getParameter("blob-key") as argument. These methods build the 
+    // shopping query and call the {@code query} method from GoogleShoppingQuerier.
 
     String shoppingQuery = getQuery(request.getParameter("photo-category"));
-    GoogleShoppingQuerier querier = new GoogleShoppingQuerier(shoppingQuery);
+    // Build the shopping query input - set language and maxResultsNumber to hard-coded values for now.
+    ShoppingQueryInput input = 
+        new ShoppingQueryInput.Builder(shoppingQuery).language("en").maxResultsNumber(20).build();
+
+    // Initialize the Google Shopping querier.
+    GoogleShoppingQuerier querier = new GoogleShoppingQuerier();
     response.setContentType("text/html");
+    
+    String shoppingQuerierResults = null;
+
     try {
-      response.getWriter().println(querier.query());
-    } catch(HttpStatusException exception) {
-      response.sendError(exception.getStatusCode(), exception.getMessage());
-    } catch(IOException ex) {
-      response.sendError(HttpServletResponse.SC_UNAUTHORIZED, "Getting results failed: " + ex.getMessage());
-    }
+      shoppingQuerierResults = querier.query(input);
+    } catch(IllegalArgumentException exception) {
+      response.sendError(500, exception.getMessage());
```

Comment:
Instead of hard-coding, it's better to use:
https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletResponse.html